tg_bot.py

```diff
@@ -260,6 +294,7 @@ def remove_job(user_id):
     # if user does not have a subscription, we want to avoid an error
     if user_id in scheduled_jobs.keys():
         scheduled_jobs.pop(user_id, None)
+        msg.reply_text('There are no active subscriptions at the moment')
```

Comment:
BTW seems like `msg` is not defined, at this point, have you tested this?

Reply:
included msg as parameter. No I didn't test, but I will with all your suggested changes.